src/libs/ajax.js

```diff
@@ -13,7 +13,7 @@ window.ajaxOverrideUtils = {
   mapJsonBody: _.curry(async (fn, res) => {
     return new Response(new Blob([JSON.stringify(fn(await res.json()))]), res)
   }),
-  makeError: _.curry(async ({ status, frequency = 1 }, res) => {
+  makeError: _.curry(({ status, frequency = 1 }, res) => {
```

Comment:
This one is meaningful, because it effectively wraps the return value in a Promise, as required by the caller. In actuality, it turns out not to break because the caller itself uses `async`, but that's just incidental.